Code/Tools/AssetProcessor/native/ui/ProductAssetDetailsPanel.cpp

```diff
@@ -46,6 +47,42 @@ namespace AssetProcessor
 
     }
 
+    void ProductAssetDetailsPanel::SetupDependencyGraph(QTreeView* productAssetsTreeView, AZStd::shared_ptr<AssetDatabaseConnection> assetDatabaseConnection)
+    {
+        m_dependencyTreeDelegate = new ProductDependencyTreeDelegate(nullptr, this);
+        m_outgoingDependencyTreeModel =
+            new ProductDependencyTreeModel(assetDatabaseConnection, m_productFilterModel, DependencyTreeType::Outgoing, this);
+        m_ui->OutgoingProductDependenciesTreeView->setModel(m_outgoingDependencyTreeModel);
+        m_ui->OutgoingProductDependenciesTreeView->setRootIsDecorated(true);
+        m_ui->OutgoingProductDependenciesTreeView->setItemDelegate(m_dependencyTreeDelegate);
+        connect(
+            productAssetsTreeView->selectionModel(), &QItemSelectionModel::selectionChanged, m_outgoingDependencyTreeModel,
+            &ProductDependencyTreeModel::AssetDataSelectionChanged);
+
+        m_incomingDependencyTreeModel =
+            new ProductDependencyTreeModel(assetDatabaseConnection, m_productFilterModel, DependencyTreeType::Incoming, this);
+        m_ui->IncomingProductDependenciesTreeView->setModel(m_incomingDependencyTreeModel);
+        m_ui->IncomingProductDependenciesTreeView->setRootIsDecorated(true);
+        m_ui->IncomingProductDependenciesTreeView->setItemDelegate(m_dependencyTreeDelegate);
+        connect(
+            productAssetsTreeView->selectionModel(), &QItemSelectionModel::selectionChanged, m_incomingDependencyTreeModel,
+            &ProductDependencyTreeModel::AssetDataSelectionChanged);
+
+        AzQtComponents::StyleManager::setStyleSheet(m_ui->OutgoingProductDependenciesTreeView, QStringLiteral("style:AssetProcessor.qss"));
+        AzQtComponents::StyleManager::setStyleSheet(m_ui->IncomingProductDependenciesTreeView, QStringLiteral("style:AssetProcessor.qss"));
+    }
+
+    
+    QTreeView* ProductAssetDetailsPanel::GetOutgoingProductDependenciesTreeView() const
```

Comment:
this is declared and implemented, but I do not see where GetOutgoingProductDependenciesTreeView() is used

Reply:
This is used in [MainWindow.cpp](https://github.com/aws-lumberyard-dev/o3de/blob/sven_ProductAssetDetail/Code/Tools/AssetProcessor/native/ui/MainWindow.cpp) for setting up and showing context menu.